src/main/java/thePackmaster/powers/maridebuffpack/HalfBlockPower.java

```diff
@@ -0,0 +1,43 @@
+package thePackmaster.powers.maridebuffpack;
+
+import com.megacrit.cardcrawl.actions.common.ReducePowerAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import thePackmaster.powers.AbstractPackmasterPower;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class HalfBlockPower extends AbstractPackmasterPower {
+    public static final String POWER_ID = makeID("HalfBlockPower");
+    public static final String NAME = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).NAME;
+    public static final String[] DESCRIPTIONS = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).DESCRIPTIONS;
+
+    public HalfBlockPower(AbstractCreature owner, int amount) {
+        super(POWER_ID,NAME,PowerType.DEBUFF,true,owner,amount);
+    }
+
+    @Override
+    public void onUseCard(AbstractCard card, UseCardAction action){
+        if (card.baseBlock >= 0){
```

Comment:
Do you know if any other base game cards do this? I think base game interactions are common enough that it would be good to hardcode them as exceptions for this power. Hopefully, this happens less with modded cards with the availability of extra magic numbers.

Reply:
Oh, Ritual Dagger actually got fixed with 2.0 it seems. No others come to mind.

Reply:
Oh nevermind, I overlooked the apply power lines. Yeah, the ID check will be necessary.